ui/app/js/controllers/rules_controller.js

```diff
@@ -321,6 +321,36 @@ function($scope, $routeParams, $location, $timeout, Rules, Search, $modal, $rout
     });
   };
 
+  $scope.openNewScheduledDuplicateModal = function(rule) {
+
+    var modalInstance = $modal.open({
+      templateUrl: 'rule_scheduled_change_modal.html',
+      controller: 'NewRuleScheduledChangeCtrl',
+      size: 'lg',
+      backdrop: 'static',
+      resolve: {
+        scheduled_changes: function() {
+          return [];
+        },
+        sc: function() {
+          sc = angular.copy(rule);
```

Comment:
This is largely correct, but you need to remove rule_id, alias and data_version from the copied version (rule_id and alias must be unique across all rules, and data_version shouldn't be provided for newly created rules).